src/library/dlgtagfetcher.ui

```diff
@@ -80,6 +80,69 @@
          </column>
         </widget>
        </item>
+      <item row="0" column="1">
+        <widget class="QLabel" name="label_4">
+         <property name="styleSheet">
+          <string notr="true">QLabel { font-weight: bold; }</string>
+         </property>
+         <property name="text">
+          <string>Current Cover Art</string>
+         </property>
+        </widget>
+      </item>
+      <item row="1" column="1">
+             <widget class="QWidget" name="verticalWidgetCover" native="true">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                <horstretch>250</horstretch>
+                <verstretch>250</verstretch>
+               </sizepolicy>
+              </property>
+              <layout class="QVBoxLayout" name="currentCoverArtLayout">
+               <property name="spacing">
+                <number>0</number>
+               </property>
+               <property name="sizeConstraint">
+                <enum>QLayout::SetDefaultConstraint</enum>
+               </property>
+              </layout>
+             </widget>
+        </item>
+      <item row="2" column="1">
+        <widget class="QLabel" name="label_5">
+         <property name="styleSheet">
+          <string notr="true">QLabel { font-weight: bold; }</string>
+         </property>
+         <property name="text">
+          <string>Found Cover Art</string>
+         </property>
+        </widget>
+      </item>
+      <item row="3" column="1">
+             <widget class="QWidget" name="verticalWidgetCover_2" native="true">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                <horstretch>250</horstretch>
+                <verstretch>250</verstretch>
+               </sizepolicy>
+              </property>
+              <layout class="QVBoxLayout" name="fetchedCoverArtLayout">
+               <property name="spacing">
+                <number>0</number>
+               </property>
+               <property name="sizeConstraint">
+                <enum>QLayout::SetDefaultConstraint</enum>
+               </property>
+              </layout>
+             </widget>
+        </item>
+      <item row="4" column="1">
+      <widget class="QPushButton" name="btnCover">
+       <property name="text">
+        <string>&amp;Set Cover Art</string>
```

Comment:
Is there a use case to use a cover form a different release than the actual release of the metadata? 
If yes, that should be probably rare. So I think I would conciser to just go with a single "Apply" button that applies all metadata at once.